build.go

```diff
@@ -51,8 +51,10 @@ func Build(entryResolver EntryResolver, installProcess InstallProcess, siteProce
 		}
 
 		logger.Process("Executing build process")
-		duration, err := clock.Measure(func() error {
-			return installProcess.Execute(context.WorkingDir, packagesLayer.Path, cacheLayer.Path)
+		var checksum string
+		duration, err := chronos.DefaultClock.Measure(func() error {
```

Comment:
In the interests of making the smallest change, I would leave the interface with an injected clock. We can have a separate conversation about the merits of removing this from the interface later.

Reply:
Not sure why this response was put here

Reply:
Please disregard the previous two answers.